tests/kernel/gen_isr_table/testcase.yaml

```diff
@@ -4,3 +4,8 @@ tests:
         stm32_min_dev_blue usb_kw24d512 v2m_beetle
     filter: CONFIG_GEN_ISR_TABLES and CONFIG_ARMV7_M_ARMV8_M_MAINLINE
     tags: interrupt isr_table
+  arch.interrupt.arc:
+    filter: CONFIG_CPU_ARCV2
+    extra_configs:
+      - CONFIG_ARC_FIRQ_STACK=y
```

Comment:
are there (other) test cases that still cover the non-FIRQ_STACK (and non-HW_STACK_PROTECTION) cases?

Reply:
if ARC_FIRQ is configured and priority 0 is used, and no firq_stack, the test will fail in build because no firq_stack.
If ARC_FIRQ  is not configured, e.g nsim_sem,  or priority 0 is not used, the test will pass.